apps/schools/domains/ticket/components/currentTicket/index.tsx

```diff
@@ -0,0 +1,276 @@
+import React, { ChangeEvent, useEffect, useState } from 'react'
+import { getUuidFromUrl } from '@domains/common/utils/getUuidFromUrl'
+import router from 'next/router'
+import { AppRoutes, RoutePath } from '@domains/common/constants/routerEnums'
+import { useOrganization } from '@domains/organization/providers/organizationProvider'
+import {
+    CLOSED_FILTER_COLOR,
+    IN_PROGRESS_FILTER_COLOR,
+    SENT_FILTER_COLOR,
+} from '@domains/ticket/components/ticketList/styles/styles'
+import { format } from 'date-fns'
+import { Checkbox, Spin, Typography } from 'antd'
+import styles from './styles/styles.module.scss'
+import { Select } from '@domains/common/components/select'
+import Image from 'next/image'
+import employee from '@public/icons/icon_employee.svg'
+import family from '@public/icons/icon_family.svg'
+import lock from '@public/icons/Lock.svg'
+import ticketSoon from '@public/image/ticketSoon.png'
+import TextArea from 'antd/lib/input/TextArea'
+import { Button } from '@domains/common/components/button'
+import classnames from 'classnames'
+import { useGetTicketQuery } from '@domains/organization/redux/organizationApi'
+import { handleQueryStatusChange } from '@domains/query/handlers/queryUpdate'
+import { QueriesTypes } from '@domains/common/redux/interfaces'
+import { useChangeStatusMutation } from '@domains/query/redux/queryApi'
+import {
+    useCreateCommentMutation,
+    useGetTicketCommentsQuery,
+    useSeenTicketCommentMutation,
+} from '@domains/ticket/redux/ticketApi'
+import { useUserProfile } from '@domains/user/providers/authProvider'
+import { handleCreateComment } from '@domains/ticket/handlers/createComment'
+import { useGetAllEmployeesQuery } from '@domains/employee/redux/employeeApi'
+
+interface CommentProps {
+    sender?: string
+    is_sender: boolean
+    familyName?: string
+    created_at?: string
+    is_internal_recipient?: boolean
+    comment: string
+}
+
+const CommentBlock: React.FC<CommentProps> = ({
```

Comment:
useMemo?
if you send one more comment there is no need to rerender old messages